src/components/timeline/VerticalIndicators.js

```diff
@@ -70,10 +70,13 @@ export class VerticalIndicators extends React.PureComponent<Props> {
         case 'DOMContentLoaded':
           color = 'var(--blue-50)';
           break;
+        case 'FirstContentfulPaint':
+        case 'FirstContentfulComposite':
+          color = 'var(--green-60)';
+          break;
         default:
-          if (marker.name.startsWith('Contentful paint ')) {
```

Comment:
`First Contentful ...` text is not another marker. It's put inside the same text marker as the description:
https://searchfox.org/mozilla-central/rev/27ddf0064aa821af8238a99621df79fb1b301c7b/layout/base/nsPresContext.cpp#2844-2845,2853

So this check was still incorrect because it was still checking `marker.name`, but this text is inside `marker.data.name`. That's why it's okay to change this directly since this was always incorrect.